File: Guess.java
import java.util.Scanner;
import java.util.Random;

public class Guess{
public static void main(String[]args){
Random rand = new Random();
Scanner input = new Scanner(System.in);
int random = rand.nextInt(1,10);
int number = 0;

while(number != random){
System.out.print("Enter a number");
number = input.nextInt();
if(number > random){
System.out.println("Number is too high");
}
if(number < random){
System.out.println("Number is too low");
}
}
if(number == random){
System.out.println("correct");}
}
}

File: Guess.py
import random
random_number = random.randint(1,10)
user_input = int(input("Enter a number: "))

while(user_input != random_number):
    if(user_input > random_number):
        print("Too high")
    else:print("Too low")
    user_input = int(input("Enter a number: "))
print("correct")
